plugins/modules/win_feature_info.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = r'''
+---
+module: win_feature_info
+short_description: Gather information about Windows features
```

Comment:
You need to add `version_added: '1.4.0'` (1.4.0 is the next release) to the documentation to denote what version this was added in.